prototype/storage_api.py

```diff
@@ -0,0 +1,175 @@
+from mmif import Mmif
+from clams import mmif_utils
+from flask import Flask, request, jsonify, send_from_directory
+from enum import Enum
+from pydantic import BaseModel
+from typing import List, Dict
+from typing_extensions import Annotated
+import os
+import yaml
```

Comment:
this yaml is used only for a limited purpose, namely to load config file. However, the existing code is based on `.env` file and `dotenv` module, so let's migrate to environment variable-based configuration to match the existing code.